src/EventTeachingService.cpp

```diff
@@ -91,26 +91,6 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
   case OPC_REQEV:
     // B2 - Read event variable in learn mode
     return handleRequestEventVariable(msg, nn, en);
-
-  case OPC_RQMN:
-    // 11 - request for node module name, excluding "CAN" prefix
-    // DEBUG_SERIAL << F("> RQMN received") << endl;
-
-    // only respond if in Learn
-    // respond with NAME
-    if (bLearn)
-    {
-      msg->len = 8;
-      msg->data[0] = OPC_NAME;
-      memcpy(msg->data + 1, controller->getModuleName(), 7);
-      controller->sendMessage(msg);
-    }
-    return PROCESSED;
-    
-  default:
-    // unknown or unhandled OPC
-    // DEBUG_SERIAL << F("> opcode 0x") << _HEX(opc) << F(" is not currently implemented")  << endl;
-    return NOT_PROCESSED;
```

Comment:
Hmm, I think you should leave in this default bit.
You should get a compile error as there isn't a return in all branches.

Reply:
But it has gone back to the way it was before.  EventTeachingService no longer needs to see OPC_RQMN.

Reply:
Sure, it is the default: part that needs to be kept. I.e. keep lines 110-113.
It shouldn't compile without those lines.

Reply:
Oh, right. I see what you mean.  Now restored.